tests/test_cli.py

```diff
@@ -1349,6 +1364,36 @@ def test_ask_sampler_kwargs_without_sampler() -> None:
         assert "`--sampler_kwargs` is set without `--sampler`." in error_message
 
 
+@pytest.mark.skip_coverage
+def test_ask_without_create_study_beforehand() -> None:
+    study_name = "test_study"
+    search_space = (
+        '{"x": {"name": "FloatDistribution", "attributes": {"low": 0.0, "high": 1.0}}, '
+        '"y": {"name": "CategoricalDistribution", "attributes": {"choices": ["foo"]}}}'
+    )
+
+    with NamedTemporaryFilePool() as tf:
+        db_url = "sqlite:///{}".format(tf.name)
+
+        args = [
+            "optuna",
+            "ask",
+            "--storage",
+            db_url,
+            "--study-name",
+            study_name,
+            "--search-space",
+            search_space,
+        ]
+
+        result = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        error_message = result.stderr.decode()
+        assert (
```

Comment:
I think that it is fragile to check the entire error message. How about avoiding to check the message?

Reply:
This testing method is commonly used in `test_cli.py`. I agree that your concern is valid, however, I think a refactoring addressing the concern is outside the scope of this PR.